### DISCUSSION TITLE
Documentation example is not working in [tabulator-tables]

### DISCUSSION THREAD

**Discussion by vikulinsd** [DefinitelyTyped/DefinitelyTyped#63523] [Issues with a @types package]
I am trying to use the examples about tooltips from the documentation @types/tabulator-tables https://tabulator.info/docs/5.4/menu
However, I get an error about return type mismatch.   As I understood the tooltip cannot be a dom node, because it is defined as `tooltip?: string | GlobalTooltipOption | undefined;` where `GlobalTooltipOption = boolean | ((event: MouseEvent, cell: CellComponent, onRender: () => void) => string);` in @types/tabulator-tables/index.d.ts. 
Moreover, if you define the callback as `as any`, the tooltips work fine. Please deal with the situation with tooltips&headerTolltips

[a comment by an automated account is not shown]

**Comment by mike-lischke** [comment]
@vikulinsd Can you share a bit code and the error you get?

**Reply by vikulinsd** [reply]
error TS2345: Argument of type '{ title: string; field: string; editable: boolean; validator: { type: (cell: CellComponent, value: any) => boolean; }[]; editor: (cell: CellComponent, onRendered: EmptyCallback, success: ValueBooleanCallback) => false | HTMLElement; ... 9 more ...; headerTooltip: (_column: ColumnComponent) => string; }' is not assignable to parameter of type 'ColumnDefinition'.
  Types of property 'tooltip' are incompatible.
    Type '(_event: MouseEvent, cell: CellComponent) => HTMLDivElement' is not assignable to type 'string | GlobalTooltipOption | undefined'.
      Type '(_event: MouseEvent, cell: CellComponent) => HTMLDivElement' is not assignable to type '(event: MouseEvent, cell: CellComponent, onRender: () => void) => string'.
        Type 'HTMLDivElement' is not assignable to type 'string'.

**Reply by mike-lischke** [reply]
How would you change the type definition to make it work correctly, instead of working around it with an any cast?
